spec/models/pds_search_result_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: pds_search_results
+#
+#  id          :bigint           not null, primary key
+#  import_type :string
+#  nhs_number  :string
+#  result      :integer          not null
+#  step        :integer          not null
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#  import_id   :bigint
+#  patient_id  :bigint           not null
+#
+# Indexes
+#
+#  index_pds_search_results_on_import      (import_type,import_id)
+#  index_pds_search_results_on_patient_id  (patient_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (patient_id => patients.id)
+#
+describe PDSSearchResult, type: :model do
+  subject(:pds_search_result) { build(:pds_search_result) }
+
+  describe "associations" do
+    it { should belong_to(:patient) }
+    it { should belong_to(:import).optional }
+  end
+
+  describe "persistence" do
+    it "persists a custom created_at when importing" do
+      timestamp = 2.days.ago
+      patient = create(:patient)
+      import = create(:cohort_import)
+
+      record =
+        described_class.new(
+          patient:,
+          import:,
+          step: :no_fuzzy_with_history,
+          result: :one_match,
+          nhs_number: "1234567890",
+          created_at: timestamp
+        )
+
+      described_class.import([record])
```

Comment:
I'm a bit confused by this bit ... we test that if the record is imported it sets the `created_at`?

Reply:
I'm just not clear on what it's testing ... it would be nice if we could test that `ProcessPatientChangesetsJob` and `CommitPatientChangesetsJob` adds the `created_at`. Normally we'd rely on it showing up in the UI in a feature test, but since the use-case here is simply that we can check the dates these records are created at as part of ops tasks, maybe it's ok to go straight to the db and check them, in a feature test. What do you think?